hackclub.com.yaml

```diff
@@ -1,4327 +1,4485 @@
 super-cdn-bot:
-  - ttl: 600
-    type: CNAME
-    value: a.selfhosted.hackclub.com.
+- octodns:
+    cloudflare:
+      proxied: true
+  ttl: 300
+  type: CNAME
+  value: a.selfhosted.hackclub.com.
 swirl:
-  - ttl: 900
-    type: CNAME
-    value: cname.vercel-dns.com.
+- ttl: 300
+  type: CNAME
+  value: cname.vercel-dns.com.
 synthax:
-  - ttl: 600
-    type: CNAME
-    value: qwel-exe.github.io.
+- ttl: 300
+  type: CNAME
+  value: qwel-exe.github.io.
 tag:
-  - ttl: 600
-    type: CNAME
-    value: cname.vercel-dns.com.
+- ttl: 300
+  type: CNAME
+  value: cname.vercel-dns.com.
 tarot:
-  - ttl: 600
-    type: CNAME
-    value: a.selfhosted.hackclub.com.
+- octodns:
+    cloudflare:
+      proxied: true
+  ttl: 300
+  type: CNAME
+  value: a.selfhosted.hackclub.com.
 tbilisi:
-  - ttl: 600
-    type: CNAME
-    value: tbilisihc.github.io.
+- ttl: 300
+  type: CNAME
+  value: tbilisihc.github.io.
 _github-pages-challenge-tbilisihc.tbilisi:
-  - ttl: 600
-    type: TXT
-    value: 352ce0458af34bcb3ac0d2f8c7920d
+- ttl: 300
+  type: TXT
+  value: 352ce0458af34bcb3ac0d2f8c7920d
 tbp:
-  - ttl: 600
-    type: NS
-    values:
-      - ns1.digitalocean.com.
-      - ns2.digitalocean.com.
-      - ns3.digitalocean.com.
+- ttl: 300
+  type: NS
+  values:
+  - ns1.digitalocean.com.
+  - ns2.digitalocean.com.
+  - ns3.digitalocean.com.
 te:
-  - ttl: 600
-    type: CNAME
-    value: cname.vercel-dns.com.
+- ttl: 300
+  type: CNAME
+  value: cname.vercel-dns.com.
 teach:
-  - ttl: 600
-    type: CNAME
-    value: hackclub.github.io.
+- ttl: 300
+  type: CNAME
+  value: hackclub.github.io.
 teenverse:
-  - ttl: 600
-    type: CNAME
-    value: teenverse.github.io.
+- ttl: 300
+  type: CNAME
+  value: teenverse.github.io.
 telemetry:
-  - ttl: 600
-    type: A
-    value: 5.161.83.132
+- ttl: 300
+  type: A
+  value: 5.161.83.132
 telecom:
-  - ttl: 600
-    type: CNAME
-    value: a.selfhosted.hackclub.com.
+- octodns:
+    cloudflare:
+      proxied: true
+  ttl: 300
+  type: CNAME
+  value: a.selfhosted.hackclub.com.
 ten-days-of-code:
-  - ttl: 600
-    type: CNAME
-    value: cname.vercel-dns.com.
+- ttl: 300
+  type: CNAME
+  value: cname.vercel-dns.com.
 testing.mail:
-  - ttl: 600
-    type: MX
-    values:
-      - exchange: mxa.mailgun.org.
-        preference: 10
-      - exchange: mxb.mailgun.org.
-        preference: 10
-  - ttl: 600
-    type: TXT
-    value: v=spf1 include:mailgun.org ~all
+- ttl: 300
+  type: MX
+  values:
+  - exchange: mxa.mailgun.org.
+    preference: 10
+  - exchange: mxb.mailgun.org.
+    preference: 10
+- ttl: 300
+  type: TXT
+  value: v=spf1 include:mailgun.org ~all
 tfss:
-  - ttl: 600
-    type: ALIAS
-    value: cname.vercel-dns.com.
-  - ttl: 600
-    type: TXT
-    values:
-      - v=spf1 include:spf.improvmx.com ~all
-  - ttl: 600
-    type: MX
-    values:
-      - exchange: mx1.improvmx.com.
-        preference: 10
-      - exchange: mx2.improvmx.com.
-        preference: 20
```

Comment:
This MX record should not have been deleted! This is a breaking change, was this communicated to the team responsible?